3rd_party/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+if (${BUILD_PYTHON})
```

Comment:
Let's rename for `externals`, that is more common name

Reply:
https://api.csswg.org/bikeshed/?force=1&url=https://raw.githubusercontent.com/vector-of-bool/pitchfork/develop/data/spec.bs